app/presenters/avatar_presenter.rb

```diff
@@ -21,6 +22,11 @@ def medium(with_default=false)
   end
 
   def large(with_default=false)
+    image_url(size: :scaled_full, fallback_to_default: with_default)
+  end
+
+  def raw(with_default=false)
+    # ToDo: Replace me with the actual raw photo.
```

Comment:
Style/CommentAnnotation: Annotation keywords like `ToDo` should be all upper case, followed by a colon, and a space, then a note describing the problem.